GameOfLifeApp/Communicator.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameOfLifeApp
+{
+    /// <summary>
+    /// Requests info from  user. Sends a message about incorrect input.
+    /// </summary>
+    public class Communicator
+    {
+        const string notANumberMessage = "Inputed value is not a number. Please input a number. Press any key to continue.";
+        const string outOfLimit = "Out of limit.";
+
+        /// <summary>
+        /// Gets data from user.
+        /// </summary>
+        /// <param name="message">Requests to provide informatiom about array size.</param>
+        /// <returns>Returns count of columns and count of rows.</returns>
+        public static string? GetInput(string message)
+        {
+            Console.Clear();
+            Console.WriteLine(message);
+            return Console.ReadLine();
+        }
+
+        /// <summary>
+        /// Checks the input and converts it to int. Requests to input a number.
+        /// </summary>
+        /// <param name="message">User`s response</param>
+        /// <returns>int</returns>
+        public static int GetNumericResponse(string message)
```

Comment:
This method doesn’t depend on certain / specific object. It is available for all. Storage efficiency. The code looks more esthetic, readable.

Reply:
It actually does depend on a component, the Console and it might not be accessible to all who will call this method. In general, its best to avoid static classes/methods, as they can be hard to test and mock. In this case, since this method is also returning a value, you would need to mock it at some point in your unit tests.

Reply:
This applies to all methods in this class as all of them are using static

Reply:
Changed a static method.